pleskdistup/common/src/util.py

```diff
@@ -10,13 +10,14 @@
 def logged_check_call(cmd: typing.Union[typing.Sequence[str], str], **kwargs) -> str:
     log.info(f"Running: {cmd!r}. Output:")
 
-    # I beleive we should be able pass argument to the subprocess function
+    # I believe we should be able pass argument to the subprocess function
     # from the caller. So we have to inject stdout/stderr/universal_newlines
     kwargs["stdout"] = subprocess.PIPE
-    kwargs["stderr"] = subprocess.STDOUT
+    kwargs["stderr"] = subprocess.PIPE
     kwargs["universal_newlines"] = True
 
     stdout = []
+    stderr = []
     process = subprocess.Popen(cmd, **kwargs)
     while None is process.poll():
```

Comment:
Why not simply use something like:
```py
    out, err = process.communicate()
```

Reply:
IIRC when we use `process.communicate`, we will be stuck until the subprocess is finished, right?

Reply:
AFAIS we're already stuck until the process terminates.
The difference would be, if we care, when the outputs are huge. I changed the logic to line process both outputs.

Reply:
We need to read the output and write it into the log file asynchronously. This is important for long-running processes like 'leapp upgrade' so we can log output while the process is still running. Some customers have used this before to confirm that the process is still active, even after the progress bar shows 100% (due to previous time estimate errors). I would like to maintain this functionality.

Reply:
Both `process.communicate()` and our current approach doesn't let the upgrader program proceed before child process it terminated. Was it already not what you expect?